xbmc/addons/InputStream.cpp

```diff
@@ -91,21 +94,22 @@ bool CInputStream::Supports(CFileItem &fileitem)
   }
 
   match = false;
+
   for (auto &path : m_pathList)
   {
     if (path.empty())
       continue;
 
-    if (fileitem.GetPath().compare(0, path.length(), path) == 0)
+    CRegExp r(true, CRegExp::asciiOnly, path.c_str());
```

Comment:
Because it is only called once at startup and only if the extension matches, my personal meaning is that time for more development is given away. The regex solution is flexible, and I don't want to think over constellations wich will arrive in the future.
But sure, if anyone has a better idea - go ahead with it!